demo/components/group-demo.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import {
+  VictoryChart, VictoryStack, VictoryBar, VictoryLine,
+  VictoryGroup, VictoryScatter, VictoryErrorBar
+} from "../../src/index";
+import { range, random } from "lodash";
+
+
+class App extends React.Component {
+
+  getGroupData() {
+    return range(5).map(() => {
+      return [
+        {
+          x: "rabbits",
+          y: random(1, 5)
+        },
+        {
+          x: "cats",
+          y: random(1, 10)
+        },
+        {
+          x: "dogs",
+          y: random(1, 15)
+        }
+      ];
+    });
+  }
+
+  getMultiData() {
+    const bars = random(3, 5);
+    return range(4).map(() => {
+      return range(bars).map((bar) => {
+        return {x: bar + 1, y: random(2, 10)};
+      });
+    });
+  }
+
+  render() {
+    const containerStyle = {
+      display: "flex",
+      flexDirection: "row",
+      flexWrap: "wrap",
+      alignItems: "center",
+      justifyContent: "center"
+    };
+
+    const chartStyle = {parent: {border: "1px solid #ccc", margin: "2%", maxWidth: "40%"}};
+
+    return (
+      <div className="demo">
+        <div style={containerStyle}>
+          <VictoryChart style={chartStyle} domainPadding={20}>
+            <VictoryStack
+              style={{
+                data: {strokeDasharray: "10, 5"}
+              }}
+              colorScale="qualitative"
+            >
+              <VictoryGroup
+                data={[
+                  {x: 1, y: 3},
+                  {x: 2, y: 4},
+                  {x: 3, y: 2}
+                ]}
+                style={{
+                  data: {width: 40, opacity: 0.6}
+                }}
+              >
+                <VictoryBar/>
+                <VictoryLine/>
+              </VictoryGroup>
+              <VictoryGroup
+                data={[
+                  {x: 1, y: 4},
+                  {x: 2, y: 5},
+                  {x: 3, y: 1}
+                ]}
+                style={{
+                  data: {width: 20, opacity: 0.8}
+                }}
+              >
+                <VictoryBar/>
+                <VictoryLine/>
+              </VictoryGroup>
+              <VictoryGroup
+                data={[
+                  {x: 1, y: 3},
+                  {x: 2, y: 2},
+                  {x: 3, y: 5}
+                ]}
+                style={{
+                  data: {width: 10, opacity: 1}
+                }}
+              >
+                <VictoryBar/>
+                <VictoryLine/>
+                <VictoryScatter
+                  symbol={"plus"}
+                  size={10}
+                  style={{
+                    data: {fill: "tomato"}
+                  }}
+                />
+              </VictoryGroup>
+            </VictoryStack>
+          </VictoryChart>
+
+          <VictoryChart style={chartStyle}>
```

Comment:
I think this chart is easier to understand, as it doesn't have the extra complication of VictoryStack.